activerecord/test/cases/adapters/postgresql/connection_test.rb

```diff
@@ -90,7 +90,7 @@ def test_reset_with_transaction
     end
 
     def test_tables_logs_name
-      @connection.tables('hello')
+      ActiveSupport::Deprecation.silence { @connection.tables('hello') }
```

Comment:
I think we should *add* test cases for `#data_sources`. Because `test_tables_logs_name` checks behavior of logging of `#tables`, and we should confirm this behavior is not regressed.